uint32 id
int16 current1
int16 current2
int16 current3
int16 current4
